src/api/index.ts

```diff
@@ -8,6 +8,7 @@ import TradeAPI from './TradeAPI';
 import NavIssuanceAPI from './NavIssuanceAPI';
 import PriceOracleAPI from './PriceOracleAPI';
 import DebtIssuanceAPI from './DebtIssuanceAPI';
+import { TradeQuoteAPI } from './utils';
```

Comment:
Mostly it's because there are a bunch of non-contract services required generate the quote. These are in different files...like GasPrice, CoinGeckoTokenList, ZeroEx etc.. 

Maybe it could be: 
```
import { UtilsAPI } from `./UtilAPI
``

```
set.utils.tradeQuote(options: QuoteOptions)
set.utils.fetchCoinPrices()
```